### PR TITLE
Snippet to generate random color code

### PR DESCRIPTION
**Pull request by arjunmahishi**
## Description

Added new snippet

## What does your PR belong to?
- [ ] Website
- [x] Snippets
- [ ] General / Things regarding the repository (like CI Integration)

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Enhancement (non-breaking improvement of a snippet)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have checked that the changes are working properly
- [x] I have checked that there isn't any PR doing the same
- [x] I have read the **CONTRIBUTING** document.

### CODE REVIEW

File: snippets/randomHexColorCode.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+### randomHexColorCode
+
+Generates a random hexadecimal color code.
+
+Use `Math.random` to generate a random number and limit that number to fall in between 0 and 16 using `Math.floor`. Use the generated random number as index to access a character from 0 to F. Append it to `color` till the length is not `7`.  
+
+```js
+const randomHexColorCode = () => {
+	  var color = '#';
```
Comment (Chalarangelo) [inline snippets/randomHexColorCode.md]: Use `let` instead of `var`.

**Comment by Joshfindit** [comment]
Quick Q: why not generate 3 random 8-bit values and concat them together as hex?

**Comment by Pl4gue** [comment]
@Joshfindit See #348.

**Comment by Joshfindit** [comment]
@Pl4gue even better 👍🏻

**Pull request merged**